trd_cron.bash

```diff
@@ -33,6 +34,13 @@ then
     exit 1
 fi
 
+if [[ ! -f "${config_file}" ]]; then
```

Comment:
Two comments, 

1. This is going to be backwards incompatible as written now, since now you require the `config_file` which didn't used to be. Would it be reasonable to fall back to defaulting to the `trd.conf` file you have in this PR if there isn't a file at `/etc/trd.conf`?
2. You might want to update the user warning to include the path that it's trying to find. Something like `"Please ensure that the configuration file (${config_file}) exists."`

Reply:
Thanks Ian.  Good thoughts:
1) Not sure how much we care about backwards incompatibility here as the volume labels were hard coded in.  If we do care about it may do something like that, or allow the user to specify an alternate config on the command line.
2) Definitely more useful for the user.  Will do.

Reply:
Fair enough. 👍